docs/docs/services/qix-engine/access-control.md

```diff
@@ -84,347 +140,378 @@ Example JWT:
-the targeted resource or resources. Otherwise returns `false`.
+This operator returns `true` only if its operands are _strictly_ not equal. String comparison is _case sensitive_ (see
+`!=` for case insensitive comparison). If one of the operands is a list, only one value in the list needs to be strictly
+unequal.
 
 **Syntax**
 
 ```c
-resource.HasPrivilege("action")
+(EXPRESSION) !== (EXPRESSION)
 ```
 
-The required parameter `action` needs to be one of the actions listed under [Actions](#actions).
-
 **Examples**
 
-_None at the moment._
+Given that `user.country` is `"uk"`:
 
-### `IsAnonymous()`
+```py
+# Evaluate to true
+user.country !== "UK"
+user.country !== {"uk", "UK", "se"}
 
-Boolean function for user conditions that returns `true` only if
-the user requesting access is logged in as anonymous.
-
-**Syntax**
-
-```c
-user.IsAnonymous()
+# Evaluate to false
+resource.org !== "uk"
+resource.org !== {"uk"}
 ```
 
-**Examples**
+#### `like`
 
-```c
-// Evaluates to `true` if the user is part of the "Developers"
-// organization, and is not anonymous:
-user.org == "Developers" and !user.IsAnonymous()
-```
+This operator provides wildcard string matching.
+
+The operator returns `true` only if the left operand matches the wildcard pattern given by the right operand.
+The comparison is _case insensitive_.
 
-### `Empty()`
+| Wildcard | Description |
+| -------- | ----------- |
+| `?`      | Matches any single character. |
+| `*`      | Matches zero or more characters. |
 
-Boolean function for resource conditions that returns `true` only if the
-specified resource has no connections (that is, has no value).
+The escape character `\` can be used to match `?`, `*`, and `\` using the escaped sequences - `\?`, `\*`, and `\\`.
 
 **Syntax**
 
-```c
-resource.resourcetype.Empty()
+```py
+(EXPRESSION) like (EXPRESSION)
 ```
 
 **Examples**
 
-_None at the moment._
+Given that `user.region` is `"us-east"` or `"us-west"`
+
+```py
+# Evaluate to true
+user.region like "us-*"
+user.region like "US-*"
+user.region like "??-*"
+
+# Evaluate to false
+user.region like "us-?"
+user.region like "uk-*"
+```
 
-### `IsOwned()`
+#### `matches`
 
-Boolean function for resource conditions that returns `true` only if the
-specified resource has an owner.
+This operator provides regular expression string matching.
+
+The operator returns `true` only if the left operand matches the regular expression given by the right operand.
 
 **Syntax**
 
-```c
-resource.IsOwned()
+```py
+(EXPRESSION) matches (EXPRESSION)
 ```
 
 **Examples**
 
-```c
-// Evaluates to `true` if the resource is owned by the user
-// being evaluated:
-resource.IsOwned() and resource.owner = user
+Suppose we want to match users that are in any of the US regions, and we want to match users that are in any of the
+number `1` or number `2` instances. A rule expressions for matching this could be:
+
+```py
+user.region matches "us-[^-]+-(1|2)"
```

Comment:
This is greedy, right? I need to say _not_ the `-` char in order for the expression to match the ending `-1` or `-2`.
Could not think of a better example. Suggestions are welcome.

Reply:
Yes that should match it